intelmq/bin/intelmq_psql_initdb.py

```diff
@@ -63,7 +68,10 @@ def generate(harmonization_file=HARMONIZATION_CONF_FILE):
         initdb += '\n    "{name}" {type},'.format(name=field, type=field_type)
 
     initdb = initdb[:-1]  # remove last ','
-    initdb += "\n);"
+    initdb += "\n);\n"
+
+    for index in INDICES:
+        initdb += 'CREATE INDEX "idx_{0}" ON events USING btree ("{0}");\n'.format(index)
```

Comment:
The name of the table should be part of the index's name in order to avoid collisions with indices on other tables. E.g. `'CREATE INDEX "idx_events_{0}" ON ...`